### PR TITLE
load-db.sh genereren in configmap

### PR DESCRIPTION
**Pull request by wschoonveld**
(ipv in google-bucket plaatsen)

### CODE REVIEW

File: helm/liwo/templates/database-deployment.yaml [hunk 1]
```diff
@@ -61,9 +65,12 @@ spec:
             - name: POSTGRES_DB
               value: "{{ .Values.database.postgres_db }}"
           volumeMounts:
-            - name: database-volume
+            - name: loaddb-volume
               mountPath: /docker-entrypoint-initdb.d
```
Comment (avgils) [inline helm/liwo/templates/database-deployment.yaml:69]: Mount for configmap should point to file in the configmap

mountPath: /docker-entrypoint-initdb.d/load-db.sh
subPath: load-db.sh

File: helm/liwo/templates/database-loaddb-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.prefix }}-{{ .Values.database.name }}-loaddb-configmap
+data:
+  load-db.sh: |-
+  #!/bin/bash
+  #set -e
```
Comment (avgils) [inline helm/liwo/templates/database-loaddb-configmap.yaml:8]: remove #set -e?

**Review by avgils — CHANGES_REQUESTED** [review]
looks good! only small comment on the configmap mount

File: helm/liwo/templates/database-deployment.yaml (hunk 1 above)
Comment (wschoonveld) [reply helm/liwo/templates/database-deployment.yaml:69]: ach ja natuurlijk

File: helm/liwo/templates/database-loaddb-configmap.yaml (hunk 2 above)
Comment (wschoonveld) [reply helm/liwo/templates/database-loaddb-configmap.yaml:8]: akkoord

**Review by avgils — APPROVED** [review]

**Pull request merged**